src/deserialization.hpp

```diff
@@ -112,7 +112,7 @@ Status deserializePredictRequest(
             auto tensorInfo = pair.second;
             auto requestInputItr = request.inputs().find(name);
             if (requestInputItr == request.inputs().end()) {
-                SPDLOG_ERROR("Failed to deserialize request. Validation of request failed");
+                SPDLOG_DEBUG("Failed to deserialize request. Validation of request failed");
```

Comment:
in this case it makes sense to either leave spdlog::debug (we loose the line but we still can debug OVMS with different CLI option.

Reply:
NVM you are changing the BUILD flags.